### ISSUE TITLE
 Ileoileal intussusception

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6407]
**Preferred term label:**
 Ileoileal intussusception

**Synonyms**
Ileoileal invagination

**Definition (free text, please give PubMed ID)**
An abnormality of the intestine in which the ileoileal part of the intestine invaginates (telescopes) into another part of the intestine.
PMID: 17872641

**Parent term (use hpo.jax.org/app)**
Intussusception

**Comment by pnrobinson** [comment]
Adding term but correcting the definition to 
A type of intussusception of the intestine in which one part of the ileum invaginates (telescopes) into another part of the ileum.
